redfish-core/lib/bios.hpp

```diff
@@ -46,6 +252,481 @@ inline void requestRoutesBiosService(App& app)
             std::bind_front(handleBiosServiceGet, std::ref(app)));
 }
 
+/**
+ * BiosSettings class supports handle GET/PATCH method for
+ * BIOS configuration pending settings.
+ */
+inline void requestRoutesBiosSettings(App& app)
+{
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/<str>/Bios/Settings")
+        .privileges({{"ConfigureManager"}})
```

Comment:
This is what we had before

Reply:
I am not understanding the CI failure actually. Does this somehow point to any merge conflict as it is showing the diff in that file?

```
[31m-            for (auto& attrItr : attrsJson.items())[m
[32m+[m[32m            for (const auto& attrItr : attrsJson.items())[m
             {[m
                 std::string attrName = attrItr.key();[m
 [m
[31m-                if (attrName == "")[m
[32m+[m[32m                if (attrName.empty())[m
                 {[m
                     BMCWEB_LOG_ERROR << "Attribute Name cannot be null";[m
                     messages::invalidObject([m

```